### DISCUSSION TITLE
[ FF7 ] Request: Button prompts for controllers

### DISCUSSION THREAD

**Discussion by SilentHeII** [julianxhokaxhiu/FFNx#345] [Ideas]
Would it be possible to add a feature that replaces the ingame generic PC prompts with controller prompts for Xbox or Playstation?

**Comment by julianxhokaxhiu** [comment]
Hi, this feature is already available through ESUI on Qhimm. There are different ways to install it, but all versions work on top of FFNx. In fact I know personally the author and we work together on bringing new enhancements. Cheers!

**Comment by SilentHeII** [comment]
While the mod certainly adds the buttons, it doesn't appear that I am able to use the stock UI as is with only the buttons replaced. Using the addon forces the enhanced UI on and I'd actually prefer the original. Thanks for the quick response.
